docs/guides/deploying.md

```diff
@@ -26,11 +26,34 @@ main()
   });
 ```
 
-You can run it with
+You can deploy in the `localhost` network following these steps:
+
+1.  Add in your `hardhat.config.js`
+
+    ```
+    networks: {
+      localhost: {
+        url: 'http://localhost:8545',
+      },
```

Comment:
This already comes in the default Hardhat config, no need to write it in your `hardhat.config.js`.

Reply:
Not if you did  `Create an empty hardhat.config.js` via the wizard. 
IMO it's clearer to have the localhost network configured so that you can understand that the `hardhat.config.js` can handle multiple networks.

Reply:
Even if you have an empty config file, that network is defined. Sorry I wasn't clear.